drivers/sensor/st/lsm6dso/lsm6dso_trigger.c

```diff
@@ -206,10 +516,11 @@ static void lsm6dso_handle_interrupt(const struct device *dev)
 #endif
 	}
 
-	gpio_pin_interrupt_configure_dt(&cfg->gpio_drdy,
-					GPIO_INT_EDGE_TO_ACTIVE);
+	gpio_pin_interrupt_configure_dt(&cfg->gpio_drdy, GPIO_INT_EDGE_TO_ACTIVE);
```

Comment:
Generally you don't want to turn on/off the edge triggers. This is outside of the scope of this PR, but I recommend revisiting this for this driver. On level interrupts you want to turn it off because it'll keep firing until you read the status register. With edge interrupts it's safe to leave it on this way you're not missing events.